### PR TITLE
Added TDP Runtime Config Options

### PR DESCRIPTION
**Pull request by krisapplegate**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?
ALL Active versions of documentation

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

File: tap-gui/runtime-config-options.hbs.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+The Tanzu CLI can be used to identify the version of Tanzu Developer portal that is available to configure. Use the following command to show the package version:
+```
+tanzu package available get tap-gui.tanzu.vmware.com/VERSION --values-schema -n INSTALL-NAMESPACE
+```
+
+Where:
+
+- `VERSION` is the version retrieved from the prior command
+- `INSTALL-NAMESPACE` is the namespace in which you have configuraed the Tanzu Application Platform installation (most often `tap-install`)
```
Comment (Braundo) [inline tap-gui/runtime-config-options.hbs.md:44]: configured

File: tap-gui/runtime-config-options.hbs.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+MAINTAINERS:            - name: VMware
+SUPPORT-DESCRIPTION:    https://tanzu.vmware.com/support
+
+  VERSION  RELEASED-AT                    
+  1.7.6    2023-10-17 00:25:21 +0000 UTC  
+```
+
+## <a id="values-schema"></a> Display the possible values options for Tanzu Developer Portal
+
+The Tanzu CLI can be used to identify the version of Tanzu Developer portal that is available to configure. Use the following command to show the package version:
```
Comment (Braundo) [inline tap-gui/runtime-config-options.hbs.md:36]: capitalize Portal

File: tap-gui/runtime-config-options.hbs.md (hunk 1 above)
Comment (RichardJJG) [reply tap-gui/runtime-config-options.hbs.md:44]: I'll fix that after merging, thanks

File: tap-gui/runtime-config-options.hbs.md (hunk 2 above)
Comment (RichardJJG) [reply tap-gui/runtime-config-options.hbs.md:36]: I'll fix that after merging, thanks

**Pull request merged**